corehq/apps/data_vault/models.py

```diff
@@ -0,0 +1,15 @@
+import secrets
+
+from django.db import models
+
+
+class VaultStore(models.Model):
```

Comment:
```suggestion
class VaultValue(models.Model):
```

Reply:
Oh, I thought VaultStore was a nice name. the model already has something called "value" as one of its columns so I think `VaultValue` wont sit well, thinking of uses
```
vault_value = VaultValue.get(pk=1)
vault_value.value
```
but rather
```
vault_vaule = VaultStore.get(pk=1).value
```

While doing more dev work, I stumbled upon using `VaultEntry` at couple of places which sounded better to me. Let me know what you think is a better name.

Reply:
`ValutEntry` seems fine

Reply:
renamed
https://github.com/dimagi/commcare-hq/pull/28161/commits/464b40200286b313842a9df30fd5e20e0dc2ecf8